test/fixtures/v2/documentGoerliValidWithDocumentStore.ts

```diff
@@ -63,3 +63,44 @@ export const documentGoerliValidWithDocumentStore: WrappedDocument<CustomDocumen
     merkleRoot: "9ece474bfd087cf3fc2e7fcf62f82af22cd8ae6a68ff40d7992b6335399309b4",
   },
 };
+
+export const documentHederaValidWithDocumentStore: WrappedDocument<CustomDocument> = {
+  version: SchemaId.v2,
+  data: {
+    $template: {
+      name: "main",
+      type: "EMBEDDED_RENDERER",
+      url: "https://tutorial-renderer.openattestation.com",
+    },
+    recipient: {
+      name: "KrypC - Piruthivi",
+      address: {
+        street: "101 ORCHARD ROAD",
+        country: "SINGAPORE",
+      },
+    },
+    issuers: [
+      {
+        name: "KrypC Issuer",
+        tokenRegistry: "0x3DE43bfd3D771931E46CbBd4EDE0D3d95C85f81A",
+        identityProof: {
+          type: "DNS-TXT",
+          location: "trustlv.org",
+        },
+      },
+    ],
+    consignment: {
+
+    },
+    declaration: {
+
```

Comment:
Can you create a new file for this fixture? dont put them together with this Goerli fixture

Reply:
it's updated plz check now